<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="homepc" ignored="0" name="Default test" tests="1" failures="1" timestamp="11 Aug 2018 05:16:33 GMT" time="0.044" errors="0">
  <testcase name="verifyCampaign" time="0.0" classname="salesforce.VerifyCampaign">
    <failure type="org.testng.TestNGException" message="
Some DataProvider public java.lang.Object[][] salesforce.SalesAnnotations.getData(java.lang.String) throws java.io.IOException parameters unresolved:  at 0 type class java.lang.String
">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public java.lang.Object[][] salesforce.SalesAnnotations.getData(java.lang.String) throws java.io.IOException parameters unresolved:  at 0 type class java.lang.String

at org.testng.internal.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:197)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:130)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyCampaign -->
</testsuite> <!-- Default test -->
